src/lib/音韻地位.ts

```diff
@@ -55,7 +58,7 @@ const 脣音分寒桓by分析體系: Record<其他分析體系, boolean> = {
   yonh: true,
 };
 
-/**  (For internal use) Indicates bypassed verification in `音韻地位.constructor` */
+/**  @internal Indicates bypassed verification in `音韻地位.constructor` */
```

Comment:
IIRC, `@internal` is a Typescript-specific tag that means "exclude its type declaration when building .d.ts", which makes sence only when it's actually exposed in the public API.

In this case, however, it's not exported to the outside, only shared within internal modules, so it won't be included in .d.ts anyways.

Reply:
Do you think I should revert this?

Reply:
> Do you think I should revert this?

I think so.

I've also just found that Typescript's doc about `@internal` mentions another TS tool called API Extractor. It also counts `@internal` as an *release tag*, meant to be included in the API, and even gives warnings if it's not exported. So, yes, this `Unchecked` shouldn't really have an API-related tag.